### PR TITLE
doc(feedback): remove beta banners and 'associated error event' mentions

### PR DESCRIPTION
**Pull request by aliu3ntry**
As of this week User Feedback has been GA'd and is the primary way of submitting feedback!

The old method, user reports, required an associated error event but this method does not. So when describing the widget/overall product, the mentions of associating events are incorrect. 

---

## Pre-merge checklist

*If you work at Sentry, you're able to merge your own PR without review, but please don't unless there's a good reason.*

- [x] Checked Vercel preview for correctness, including links
- [x] PR was reviewed and approved by any necessary SMEs
- [x] PR was reviewed and approved by a member of the [Sentry docs team](https://github.com/orgs/getsentry/teams/docs)

## Description of changes

*Describe your changes here. If your PR relates to or resolves an issue, add a link to that too.*

## Legal Boilerplate

Look, I get it. The entity doing business as "Sentry" was incorporated in the State of Delaware in 2015 as Functional Software, Inc. and is gonna need some rights from me in order to utilize my contributions in this here PR. So here's the deal: I retain all rights, title and interest in and to my contributions, and by keeping this boilerplate intact I confirm that Sentry can use, modify, copy, and redistribute my contributions, under Sentry's choice of terms.

## Extra resources

- [Sentry Docs contributor guide](https://docs.sentry.io/contributing/)

### CODE REVIEW

File: includes/feature-stage-beta-user-feedback.mdx [hunk 1]
```diff
@@ -1,5 +0,0 @@
-<Note title="A Wild User Feedback Appears!">
-
-Sentry released a new version of User Feedback that supports collecting feedback at any time from your users, without requiring an error event. It is currently in open beta and subject to change.
```
Comment (michellewzhang) [inline includes/feature-stage-beta-user-feedback.mdx:3]: i wonder if we can re-utilize this to say UF is in GA now

Comment (aliu3ntry) [reply includes/feature-stage-beta-user-feedback.mdx:3]: oh a banner for that could be nice! Maybe add in a new file + PR though?

**Review by jas-kas — APPROVED** [review]

**Comment by jas-kas** [comment]
Looks good to me! Checked the Vercel preview for both User Feedback pages in docs. :)

File: includes/feature-stage-beta-user-feedback.mdx (hunk 1 above)
Comment (jas-kas) [reply includes/feature-stage-beta-user-feedback.mdx:3]: We could :o Not something we've done in the past for docs though, so I'd make sure to get the docs teams eyes on it for their opinion

File: docs/platforms/javascript/common/user-feedback/index.mdx [hunk 2]
```diff
@@ -1,11 +1,9 @@
 ---
 title: Set Up User Feedback
-description: "Learn more about collecting user feedback when an event occurs. Sentry pairs the feedback with the original event, giving you additional insight into issues."
+description: "Learn how to enable User Feedback in your app if it is not already set up."
```
Comment (vivianyentran) [inline docs/platforms/javascript/common/user-feedback/index.mdx:3]:
```suggestion
description: "Learn how to enable User Feedback in your app."
```

**Review by vivianyentran — APPROVED** [review]
Thanks for keeping the docs updated!

**Pull request merged**